package.json

```diff
@@ -14,15 +14,19 @@
   "dependencies": {
     "@expo-google-fonts/didact-gothic": "^0.1.0",
     "@expo-google-fonts/poppins": "^0.1.0",
+    "@react-native-async-storage/async-storage": "1.13.4",
     "@react-native-community/masked-view": "0.1.10",
     "@react-navigation/native": "5.9.2",
     "@react-navigation/stack": "5.14.2",
+    "aes-js": "3.1.2",
+    "bcryptjs": "2.4.3",
```

Comment:
cleanup the packages if we're not using bcrypt

Reply:
Done!